Sources/LiveKit/Track/Local/LocalAudioTrack.swift

```diff
@@ -42,15 +42,15 @@ public class LocalAudioTrack: LocalTrack, AudioTrack {
     }
 
     @discardableResult
-    override func start() -> Promise<Void> {
-        super.start().then(on: .sdk) {
+    override func publish() -> Promise<Void> {
```

Comment:
is this a function that's exposed to users?

Reply:
no, this is internal only